brush/src/providers/ROMProvider.js

```diff
@@ -1,14 +1,47 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import PropTypes from 'prop-types'
+import localforage from 'localforage'
 import ROMContext from '../context/ROMContext'
 
+const useROMBuffer = () => {
+  const romBufferStates = {
+    empty: { memory: null, status: 'empty' },
+    loaded: memory => ({ memory, status: 'loaded' }),
+    starting: { memory: null, status: 'starting' },
+  }
+
+  const [romBuffer, setROMBuffer] = useState(romBufferStates.starting)
```

Comment:
Don't you think setROMBufferState is a better name? I had to struggle some few minutes before understanding what was being set.

Reply:
@vinibern5 Make sense... Well... then we'll need change some others names. A pattern in react hooks is that the first variable is `foo` and the second variable is `setFoo`, for example.